jwst/background/background_sub.py

```diff
@@ -42,10 +42,10 @@ def background_sub(input_model, bkg_list, sigma, maxiters):
     result = subtract_images.subtract(input_model, bkg_model)
 
     # Close the average background image and update the step status
```

Comment:
Suggest removing the commented code completely; the model is being returned and there should not be even a hint of closing it.